### ISSUE TITLE
first and second round?

### ISSUE THREAD

**Issue by FabianRoger** [FabianRoger/AoB_Butterflies#14]
It is not clear from the data what is the 'first round' and what is the 'second round'. Below I visualize the pattern in the data. I looked at the years each site id has been visited and summarised the visiting pattern in unique visiting patters & counted how many siteIDs had this visiting pattern. 

Most Site IDs seems to have been visited three times, some more often and some only once. I am not sure how I am supposed to subdivide this into two rounds. 

![image](https://user-images.githubusercontent.com/5137092/105013903-05fff780-5a40-11eb-9de0-9d6ce735c736.png)

**Comment by larspett** [comment]
The first round is 2009-2013, the second 2014-2018. But there is like you say a lot of variation. But lets just split it into these two periods

**Comment by FabianRoger** [comment]
hmmm... ok. 

This is what it will look like

![image](https://user-images.githubusercontent.com/5137092/105050710-5a21d080-5a6e-11eb-8ebd-44b90d8e412c.png)

**Comment by larspett** [comment]
Yes, that is what they ask for so lets keep it that way
